cyences_app_for_splunk/default/macros.conf

```diff
@@ -383,12 +383,12 @@ definition = 10
 
 # Network
 [cs_network_ddos_min_m_packets]
-# no. of buckets in millions
+# no. of packets in millions
 definition = 0.5
 
 [cs_network_outbound_min_m_packets]
-# no. of buckets in millions
-definition = 0.5
+# no. of packets in millions
+definition = <TODO>
```

Comment:
What should be the value here?

Reply:
On cerberos, out of 6150 src_ips 45 src_ips have more than 1 million packet transfer for outbound. May be we can set it to 1M.

![image](https://github.com/user-attachments/assets/a7faf8f4-4406-4a32-a0eb-73ead270fd01)

Reply:
If this is hourly then 1 million bottom limit looks okay